Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -25,11 +25,11 @@ struct PlaceUtilities {
         return categories?.prefix(MaxDisplayedCategories).joined(separator: " • ")
     }
 
-    static func updateReviewUI(fromProvider provider: ReviewProvider?, onView view: ReviewContainerView) {
+    static func updateReviewUI(fromProvider provider: ReviewProvider?, onView view: ReviewContainerView, isTextShortened: Bool = false) {
         guard let provider = provider else {
             setSubviewAlpha(0.4, forParent: view)
             view.score = nil
-            view.numberOfReviewersLabel.text = "No data available"
+            view.numberOfReviewersLabel.text = "No data" + (isTextShortened ? "" : " available")
```

Comment:
I don't think we need to have the flag. Just shorted the text. I don't think there is a situation where we are going to want the whole text.